org.rncbc.qsynth.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "org.rncbc.qsynth",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "6.2",
+  "sdk": "org.kde.Sdk",
+  "command": "qsynth",
+  "rename-icon": "qsynth",
+  "rename-desktop-file": "qsynth.desktop",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* Note playback */
+    "--socket=pulseaudio",
+    /* MIDI */
+    "--device=all",
+    /* Allow loading, saving files from anywhere (portals don’t work yet) */
+    "--filesystem=home",
+    /* Allow other instances to see lockfiles */
+    "--env=TMPDIR=/var/tmp",
+    /* On SuSE it seems to be necessary for sound to work */
+    "--env=ALSA_CONFIG_PATH="
+  ],
+  "cleanup": [
+    "/share/man",
+    "/share/doc",
+    "*.la"
+  ],
+  "modules": [
+    "shared-modules/linux-audio/jack2.json",
+    "shared-modules/linux-audio/libinstpatch.json",
+    "shared-modules/linux-audio/fluidsynth2.json",
+    {
+      "name": "qsynth",
+      "buildsystem": "cmake-ninja",
+      "post-install": [
+        "install src/appdata/qsynth.appdata.xml -D /app/share/metainfo/${FLATPAK_ID}.xml"
```

Comment:
instead you should use a `rename-appdata-file: qsynth.appdata.xml` at the top since it's installed (from reading the source)

Reply:
also it will need a patch for `release` and `content_rating` that are missing.